requirements.txt

```diff
@@ -21,12 +21,13 @@ django-filer>=1.3
 Django<3.1
 html5lib>=1.0.1
 Pillow>=3.0
-six
+six>=1.15.0
 pytz>=2020.1
 gunicorn
 requests>=2.24.0
 django-pipeline>=2.0.5,<3
 psycopg2-binary>=2.8.5,<3
 django-better-admin-arrayfield>=1.3.0, <2
 celery>=4.4.0, <5
-django-celery-results>=1.2.1, <2
```

Comment:
Is there a reason why you remove django-celery-results?

Reply:
no, I think misclick. I'll add it back in